packages/cli/src/lib/platform/common.ts

```diff
@@ -36,14 +36,17 @@ export const getMiddlewareClient = (): MiddlewareClient => {
 };
 
 export const setPlatformDefaults = async () => {
+    const middlewareClient = getMiddlewareClient();
+    const middlewareManager = await middlewareClient.getManagers();
```

Comment:
managers

Reply:
and can be reused at line 51 without requesting MW second time

Reply:
fixed with [91f7afc](https://github.com/scramjetorg/transform-hub/pull/883/commits/91f7afc83a4bfcf7ba2aaf7c67c5c047af3abf9d)